apps/vs-code-designer/src/app/commands/workflows/unitTest/runUnitTest.ts

```diff
@@ -13,15 +13,63 @@ import * as path from 'path';
 import { getWorkspacePath, isMultiRootWorkspace } from '../../../utils/workspace';
 import { getLatestBundleVersion } from '../../../utils/bundleFeed';
 import { activateAzurite } from '../../../utils/azurite/activateAzurite';
+import type { TestFile } from '../../../tree/unitTestTree/testFile';
 import type { UnitTestExecutionResult } from '@microsoft/vscode-extension-logic-apps';
 
 /**
  * Runs a unit test for a given node in the Logic Apps designer.
+ * @param context - The action context.
+ * @param node - The URI representing the node to run the unit test for.
+ */
+export async function runUnitTest(context: IActionContext, node: vscode.Uri): Promise<void> {
+  let unitTestPath: string;
+  if (node && node instanceof vscode.Uri) {
+    unitTestPath = node.fsPath;
+  } else if (isMultiRootWorkspace()) {
+    const testsDirectory = getTestsDirectory(vscode.workspace.workspaceFolders[0].uri.fsPath);
+    const unitTest = await pickUnitTest(context, testsDirectory.fsPath);
+    unitTestPath = (vscode.Uri.file(unitTest.data) as vscode.Uri).fsPath;
+  } else {
+    throw new Error(localize('expectedWorkspace', 'In order to create unit tests, you must have a workspace open.'));
```

Comment:
NIT: Can you update the text over here? Instead of `create` maybe `run`? This might be a copy/paste mistake from me

Reply:
whoops, just updated